src/components/atoms/AnywayMostSevereAccidentsMarker.tsx

```diff
@@ -1,15 +1,52 @@
 import React, { FC, useState } from 'react';
+import { makeStyles } from '@material-ui/core';
 import L from 'leaflet';
 import { Marker, Popup } from 'react-leaflet';
 import MapIcon from '../atoms/AnywayMapIcon';
 import TooltipMarker, { TooltipOffset } from './TooltipMarker';
-
+import { AnyWayButton } from '../atoms/AnyWayButton';
+import TooltipArrow, { ArrowDirection } from './TooltipArrow';
 interface IProps {
   data: any;
   tooltipOffset: TooltipOffset;
 }
 
+const useStyles = makeStyles({
+  root: {
+    '& .leaflet-popup-content-wrapper': {
+      borderRadius: 6,
+      backgroundColor: '#e2e2e2f0',
```

Comment:
same as above - externalize to global as a new color in pallet. name variable as `lightBackgroundColor`